app/talk/active-users.spec.js

```diff
@@ -11,13 +11,12 @@ const users = [
 ];
 
 describe('ActiveUsers', function () {
+  const getActiveIdsStub = sinon.stub(ActiveUsers.prototype, 'getActiveUserIds').callsFake(() => Promise.resolve(activeIds));
+  const fetchUsersSpy = sinon.spy(ActiveUsers.prototype, 'fetchUncachedUsers');
+  const pageCountSpy = sinon.spy(ActiveUsers.prototype, 'pageCount');
+  const boundedPageSpy = sinon.spy(ActiveUsers.prototype, 'boundedPage');
+  const userIdSpy = sinon.spy(ActiveUsers.prototype, 'userIdsOnPage');
```

Comment:
Same comment about putting these in a `before` block and adding an `after` one to restore these.